### PR TITLE
update(worker): add origin note id 

### PR DESCRIPTION
**Pull request by polebug**
- add origin note id 
- recover lens indexer

### CODE REVIEW

File: service/indexer/internal/server/server.go [hunk 1]
```diff
@@ -165,6 +173,7 @@ func (s *Server) Initialize() (err error) {
 		gitcoin.New(),
 		snapshot.New(),
 		crossbell.New(),
+		lens_worker.New(),
```
Comment (kallydev) [inline service/indexer/internal/server/server.go:176]: ?

File: service/indexer/internal/datasource/pregod_etl/lens/datasource.go [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+	var err error
+	defer func() { opentelemetry.Log(trace, message, profileID, err) }()
+
+	ethclient, err := ethclientx.Global(message.Network)
+	if err != nil {
+		return nil, err
+	}
+
+	// rpc
+	iLensHub, err := contract.NewILensHub(lens.HubProxyContractAddress, ethclient)
```
Comment (kallydev) [inline service/indexer/internal/datasource/pregod_etl/lens/datasource.go:117]: block number

**Review by kallydev — CHANGES_REQUESTED** [review]

File: service/indexer/internal/server/server.go (hunk 1 above)
Comment (polebug) [reply service/indexer/internal/server/server.go:176]: what is problem

File: service/indexer/internal/datasource/pregod_etl/lens/datasource.go (hunk 2 above)
Comment (polebug) [reply service/indexer/internal/datasource/pregod_etl/lens/datasource.go:117]: unnecessary

**Review by kallydev — APPROVED** [review]

**Pull request merged**